elections/steering/2024/README.md

```diff
@@ -85,6 +85,15 @@ Corporate affiliation is applied after the election. If an organization finds
 itself with too many representatives it is up to those individuals to come
 to a consensus on who should serve on the committee.
 
+Contributors may check their voter eligibility at any time once the election
+process starts, by going to the [election app], logging in, navigating to
+the current year election, and seeing if the screen there says that they are eligible.
+That screen takes its data from the [voters.yaml] file.
+
+If the app does not say that you are eligible, because you have worked on
+Kubernetes in a way that is NOT reflected in GitHub contributions, you can use
+the exception form built into the same app to ask to participate in the election.
```

Comment:
```suggestion
the exception form built into the same app to ask to participate in the election.  See Voter Exception below for how this works.
```

... should probably let folks know to continue reading.

Reply:
Solved!